mergin/client.py

```diff
@@ -306,6 +306,23 @@ def username(self):
             return None  # not authenticated
         return self._user_info["username"]
 
+    def user_service(self):
```

Comment:
I suggest to add a test even is such trivial case (e.g. to cover issue when url changes on server).